eclipsePlugin/doc/building_spotbugs_plugin.txt

```diff
@@ -41,9 +41,10 @@ Import the projects
 8) Choose [File | Import General | Existing Projects into Workspace | Next]. The Import project import wizard opens.
 9) In the wizard, proceed as follows:
    a) Choose [Select root directory].
-   b) Enter ~/git/spotbugs. Click [Browse]
-   c) Click [Finish].
-   g) Now "spotbugs" project will be compiled by Eclipse and should not have any errors,
+   b) Enter ~/git/spotbugs. Click [Browse]. Make sure the checkbox "Search for nested projects" is selected.
```

Comment:
@sewe It depends on your point of view. The document is titled "Building spotbugs plugin" (and lives in the Eclipse plugin directory), which makes me think the original author only had the Eclipse plugin in mind. I guess you contribute to both the plugin and the main project. I myself see the main project just as a dependency for the plugin, and basically don't care about it. :)
However, of course feel free to make it work for both cases. My change just assures it works _at least_ for someone who wants to build the eclipse plugin (since it did not work at all with the old description).